backend/tessen/config.go

```diff
@@ -0,0 +1,34 @@
+package tessen
+
+import (
+	"fmt"
+)
+
+const (
+	// LowerBound is the minimum frequency that tessen will phone home
+	LowerBound = 1
+
+	// UpperBound is the maximum frequency that tessen will phone home
+	UpperBound = 360
+
+	// DefaultTessenFrequency is the default frequency at which tessen will phone home
+	DefaultTessenFrequency = 1
+)
+
+// Config is the tessen configuration
+type Config struct {
+	OptOut bool `json:"opt_out"`
+}
+
+// ValidateFrequency returns an error if the tessen frequency is not within the upper and lower bound limits
+func ValidateFrequency(freq uint32) error {
+	if !(freq >= LowerBound && freq <= UpperBound) {
+		return fmt.Errorf("tessen frequency must be in between the lower and upper bound limits (%d-%d)", LowerBound, UpperBound)
+	}
+	return nil
+}
+
+// DefaultConfig returns the default tessen configuration
+func DefaultConfig() *Config {
+	return &Config{}
```

Comment:
Should this be populated with default values?

Reply:
The default value of the `OptOut` field is already false.. unless the empty struct is not programmatically equivalent?

Reply:
I was thinking about the frequency values

Reply:
Frequency is hardcoded at once per minute, and a user cannot modify it. The frequency can only be modified or overwritten if the Tessen service returns a new frequency within the header. I figured the current frequency would be a field belonging to the Tessen daemon. Does that make sense?